README.md

```diff
@@ -200,6 +200,46 @@ Or, to format only specified files, use
 nox -s format -- example.py pygdbmi/IoManager.py
 ```
 
+### Making a release
+
+Only maintainers of the [pygdbmi package on PyPi](https://pypi.org/project/pygdbmi/) can make a release.
+
+In the following steps, replace these strings with the correct values:
+
+- `<REMOTE>` is the name of the remote for the main pygdbmi repository (for instance, `origin`)
+- `<VERSION>` is the version number chosen in step 2.
+
+To make a release:
+
+1. Checkout the `master` branch and pull from the main repository with `git pull <REMOTE> master`
+2. Decide the version number for the new release:
```

Comment:
See my email about version numbers.

Reply:
Prettymuch all of my projects follow semver with the leading 0 prefix. I guess I've had an aversion to making a 1.0 release because it somehow means the project is finished or something, but admittedly I don't really have a good reason for avoiding it. I guess I don't like setting expectations too high :). If you want to change it to 1.0.0 for this release I'm fine with it.

Reply:
I don't feel strongly either way so, as your other projects are semver with `0.` we can keep that.